celery/app/builtins.py

```diff
@@ -44,7 +44,7 @@ def add_unlock_chord_task(app):
     from celery.exceptions import ChordError
     from celery.result import allow_join_result, result_from_tuple
 
-    @app.task(name='celery.chord_unlock', max_retries=None, shared=False,
+    @app.task(name='celery.chord_unlock', max_retries=10, shared=False,
```

Comment:
I'm not sure if this a breaking change or not? can you please also add/update unit tests and integration tests for this?